tests/cms/views/poi_categories/test_poi_category_form_view.py

```diff
@@ -0,0 +1,471 @@
+                        "appointmentOnly": False,
+                        "timeSlots": [],
+                    },
+                    {
+                        "allDay": False,
+                        "closed": True,
+                        "appointmentOnly": False,
+                        "timeSlots": [],
+                    },
+                    {
+                        "allDay": False,
+                        "closed": True,
+                        "appointmentOnly": False,
+                        "timeSlots": [],
+                    },
+                    {
+                        "allDay": False,
+                        "closed": True,
+                        "appointmentOnly": False,
+                        "timeSlots": [],
+                    },
+                ]
+            ),
+            "initial-opening_hours": json.dumps(
+                [
+                    {
+                        "allDay": False,
+                        "closed": True,
+                        "appointmentOnly": False,
+                        "timeSlots": [],
+                    },
+                    {
+                        "allDay": False,
+                        "closed": True,
+                        "appointmentOnly": False,
+                        "timeSlots": [],
+                    },
+                    {
+                        "allDay": False,
+                        "closed": True,
+                        "appointmentOnly": False,
+                        "timeSlots": [],
+                    },
+                    {
+                        "allDay": False,
+                        "closed": True,
+                        "appointmentOnly": False,
+                        "timeSlots": [],
+                    },
+                    {
+                        "allDay": False,
+                        "closed": True,
+                        "appointmentOnly": False,
+                        "timeSlots": [],
+                    },
+                    {
+                        "allDay": False,
+                        "closed": True,
+                        "appointmentOnly": False,
+                        "timeSlots": [],
+                    },
+                    {
+                        "allDay": False,
+                        "closed": True,
+                        "appointmentOnly": False,
+                        "timeSlots": [],
+                    },
+                ]
+            ),
+            "appointment_url": "",
+            "category": str(id_of_unused_poicategory),
+            "icon": "",
+        },
+    )
+
+    delete_poicategory_url = reverse(
+        "delete_poicategory", kwargs={"pk": id_of_unused_poicategory}
+    )
+    response = client.post(delete_poicategory_url, data={})
+    redirect = response.headers.get("location")
+    assert response.status_code == 302
+    response = client.get(redirect)
+
+    amount_of_poicategories = POICategory.objects.count()
```

Comment:
In other tests in this file this variable is capitalized. Pep8 is not really helpful whether it should be (https://peps.python.org/pep-0008/#constants) but as I understand it, constants should not be defined on the function level. So it is probably best to use snake_case for all these variables (?)